### PR TITLE
Cake 5678 - Create new Campaign for HBO Max in Mobile and Desktop in-content Affiliate units

### PR DESCRIPTION
**Pull request by unijuglr**
## Links

* https://wikia-inc.atlassian.net/browse/CAKE-5678

## Description
Adding config and a slight template addition to support HBO MAX campaign
Can go live without concern, because it will be controlled by db rows being added at launch time

## Reviewers

Use `@` mentions here. Alternatively, assign the pull request to a person.
@Wikia/cake

### CODE REVIEW

File: app/services/affiliate-slots-units.json [hunk 1]
```diff
@@ -1,4 +1,24 @@
 [
+  {
+    "campaign": "hbomax",
+    "category": "hbomax",
+    "country": ["US"],
+    "heading": "Where HBO Meets So Much More",
+    "subheading": "START YOUR FREE TRAIL",
```
Comment (cschucode) [inline app/services/affiliate-slots-units.json]: TRAIL or TRIAL?

File: app/services/affiliate-slots-units.json [hunk 2]
```diff
@@ -1,4 +1,24 @@
...
+    "link": "https://www.hbomax.com/gzm",
+    "image": "https://static.wikia.nocookie.net/2ef28d1f-a27b-4ba3-943e-33948fddd089",
+    "extra_disclaimer": "Free trail for new customers only. Restrictions Apply."
```
Comment (cschucode) [inline app/services/affiliate-slots-units.json]: (see above)

File: app/services/affiliate-slots-units.json [hunk 3]
```diff
@@ -1,4 +1,24 @@
...
+  },
+  {
```
Comment (cschucode) [inline app/services/affiliate-slots-units.json:12]: Did this get added a second time?

File: app/services/affiliate-slots-units.json [hunk 4]
```diff
@@ -1,4 +1,24 @@
...
+    "campaign": "ddb",
+    "category": "ddb-test-unit-id",
```
Comment (cschucode) [inline app/services/affiliate-slots-units.json]: Looks like we already have this unit.

**Review by cschucode — CHANGES_REQUESTED** [review]
Requested:
* fix typos
* remove extra unit

File: app/services/affiliate-slots-units.json (hunk 4 above)
Comment (unijuglr) [reply app/services/affiliate-slots-units.json]: weird...

**Comment by cschucode** [comment]
Are we going to need a big unit as well? There should be two for every category/campaign. One with the "isBig" set to `true` and one without it for the `post-search-results` unit?

**Review by cschucode — APPROVED** [review]
LGTM.

**Pull request merged**